certified-connectors/Maximizer/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5059 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Maximizer",
+    "description": "Maximizer is an integration for Maximier CRM to CRUD Maximizer entries.",
+    "version": "1.202109281545",

```

Comment:
Hi @MaximizerIntegration, Thank you so much for your submission. 
May I ask you please to change the version to "1.0" since this is a new connector.